<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(83, 83, 83, 80);
selection-color: rgb(255, 19, 11);
color: rgb(255, 128, 0);
selection-color: rgb(255, 148, 17);
alternate-background-color: rgb(255, 192, 121);
border-color: rgb(255, 14, 14);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCommandLinkButton" name="commandLinkButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;System&quot;;</string>
    </property>
    <property name="text">
     <string>New password</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="changeMasterKey">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;System&quot;;</string>
    </property>
    <property name="text">
     <string>Change master-key</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>200</y>
      <width>211</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>170</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>New key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveMasterKey">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>240</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>351</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-qqw-Desktop_Qt_5_15_0_MSVC2015_64bit-Debug/sized.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="lblshortcut">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>90</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Generated password:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="generatedLbl">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>140</y>
      <width>421</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="qrGenBth">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>20</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 166, 12);
color: rgb(0, 0, 0);
font: 700 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>My qr</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="generate_passwdBth">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;System&quot;;</string>
    </property>
    <property name="text">
     <string>Generate passwords</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="viewPasswdsBth">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>183</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;System&quot;;</string>
    </property>
    <property name="text">
     <string>View passwords</string>
    </property>
   </widget>
   <widget class="QFrame" name="passwdFrame">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>361</width>
      <height>341</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="passPoint">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Password:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passLine">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="urlPoint">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;URL/Alias:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="urlLine">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="saveBth">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>240</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="usersPasswdTedit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>541</width>
      <height>471</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 11pt &quot;Tahoma&quot;;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuddddd">
    <property name="title">
     <string>Passwords</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionDelete_all_Passwords"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionVersion_3_9"/>
    <addaction name="actionjprogram_bk_com"/>
   </widget>
   <addaction name="menuddddd"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDelete_all_Passwords">
   <property name="text">
    <string>Remove all Passwords</string>
   </property>
  </action>
  <action name="actionVersion_3_9">
   <property name="text">
    <string>Version: 4.2</string>
   </property>
  </action>
  <action name="actionjprogram_bk_com">
   <property name="text">
    <string>Dev: GemProgram</string>
   </property>
  </action>
  <action name="actionNew_Password">
   <property name="text">
    <string>New Password</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionHome_Wi_Fi">
   <property name="text">
    <string>Home Wi-Fi</string>
   </property>
  </action>
  <action name="actionOthers">
   <property name="text">
    <string>Others</string>
   </property>
  </action>
  <action name="actionReestablish">
   <property name="text">
    <string>Reestablish</string>
   </property>
  </action>
  <action name="actionStart_over">
   <property name="text">
    <string>Start over </string>
   </property>
  </action>
  <action name="actionRegenerate_qr">
   <property name="text">
    <string>Regenerate qr</string>
   </property>
  </action>
  <action name="actionWrite_support">
   <property name="text">
    <string>Write support</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
